### PR TITLE
add resource request

### PR DESCRIPTION
**Pull request by klawani**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

add resource request to test-service

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x ] No
```

### CODE REVIEW

File: java/templates/tests/test-service.yaml [hunk 1]
```diff
@@ -12,4 +12,11 @@ spec:
       - name: SERVICE_NAME
         value: {{ template "hmcts.java.releaseName" . }}
     command: ["sh", "-c", "httpstatuscode=$(wget -S http://$SERVICE_NAME/health 2>&1 | grep HTTP/ | awk 'END{print $2}') && [ \"$httpstatuscode\" = \"200\" ]"]
-  restartPolicy: Never
\ No newline at end of file
+    resources:
+      limits:
+        cpu: 2500m
```
Comment (timja) [inline java/templates/tests/test-service.yaml:17]: these limits seem very excessive
```suggestion
        cpu: 500m
```

File: java/templates/tests/test-service.yaml [hunk 2]
```diff
@@ -12,4 +12,11 @@ spec:
...
+        memory: 1Gi
+      requests:
+        cpu: 25m
+        memory: 512Mi
```
Comment (timja) [inline java/templates/tests/test-service.yaml:21]:
```suggestion
        memory: 128Mi
```

File: java/templates/tests/test-service.yaml [hunk 3]
```diff
@@ -12,4 +12,11 @@ spec:
       - name: SERVICE_NAME
         value: {{ template "hmcts.java.releaseName" . }}
     command: ["sh", "-c", "httpstatuscode=$(wget -S http://$SERVICE_NAME/health 2>&1 | grep HTTP/ | awk 'END{print $2}') && [ \"$httpstatuscode\" = \"200\" ]"]
-  restartPolicy: Never
\ No newline at end of file
+    resources:
+      limits:
+        cpu: 2500m
+        memory: 1Gi
```
Comment (timja) [inline java/templates/tests/test-service.yaml:18]:
```suggestion
        memory: 512Mi
```

File: java/templates/tests/test-service.yaml (hunk 1 above)
Comment (klawani) [reply java/templates/tests/test-service.yaml]: done

File: java/templates/tests/test-service.yaml (hunk 3 above)
Comment (klawani) [reply java/templates/tests/test-service.yaml]: done

File: java/templates/tests/test-service.yaml (hunk 2 above)
Comment (klawani) [reply java/templates/tests/test-service.yaml:18]: done

**Pull request merged**